### PR TITLE
bugfix: epoch test mock leaks

### PR DESCRIPTION
**Pull request by topocount**
Epoch test mocks are borked in main currently. I'm honestly not sure why
this issue wasn't tripped in the topic branch during tests, but now jest
mock state is leaking between tests, causing call traces to show up
across multiple tests. This is fixed by clearing the mocks before each
test.

test plan:

Ensure unit tests are passing in CI and locally if you want. This was
broken locally for me, as well as in main, once i pulled these changes
down after the PR was merged.

### CODE REVIEW

File: api/hasura/cron/_epochs.test.ts [hunk 1]
```diff
@@ -643,7 +647,7 @@ describe('epoch Cron Logic', () => {
       const input = getEpochInput('notifyStart', {});
       const result = await notifyEpochStart(input);
       expect(result).toEqual([]);
-      expect(sendSocialMessage).not.toBeCalled();
+      //expect(mockSendSocial).not.toBeCalled();
```
Comment (gregpassmore) [inline api/hasura/cron/_epochs.test.ts:650]: just curious why this one is commented out

Comment (topocount) [reply api/hasura/cron/_epochs.test.ts:650]: Ah thank you! Will fix

**Comment by CryptoGraffe** [comment]
looks good to me

**Review by levity — APPROVED** [review]

**Review by CryptoGraffe — APPROVED** [review]
lg2m

**Pull request merged**